pubspec.yaml

```diff
@@ -2,14 +2,14 @@ name: json_rpc_2
 version: 2.0.9
 author: Dart Team <misc@dartlang.org>
 description: An implementation of the JSON-RPC 2.0 spec.
```

Comment:
A bit longer? Explain if this supports client and/or server?

https://pub.dartlang.org/packages/json_rpc_2#-analysis-tab-

Reply:
Sure